mautic/mautic
# Fix: Froala editor: useClassess to false

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Just set useClassess option to false.
For this option to work correctly, it is necessary to load the CSS files from the same domain the editor is running on.
https://www.froala.com/wysiwyg-editor/docs/options#useClasses

Another option is uncommented  these lines
https://github.com/mautic/mautic/blob/d3aebf3644ad423614f0e33093e5b4f2bacfce2d/app/bundles/CoreBundle/Assets/css/libraries/libraries.less#L31
https://github.com/mautic/mautic/blob/d3aebf3644ad423614f0e33093e5b4f2bacfce2d/app/bundles/CoreBundle/Assets/css/libraries/builder.less#L5

I didn't know why it was commented. Should somebody from  core team should know


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Create form and create description area
2. Paste image and try set align to right
3. Nothing happend

#### Steps to test this PR:
1.  Apply PR and run `app/console m:a:g`
2.  Create form and create description area
3. Paste image and try set align to right
4. Should work properly

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
